src/main/kotlin/no/nav/familie/ef/sak/tilbakekreving/TilbakekrevingService.kt

```diff
@@ -60,4 +77,50 @@ class TilbakekrevingService(private val tilbakekrevingRepository: Tilbakekreving
         return tilbakekrevingRepository.existsById(behandlingsId)
     }
 
+    fun hentBrev(behandlingId: UUID, varseltekst: String): ByteArray {
+        validerIkkeFerdigstiltBehandling(behandlingId)
+        val fagsak = fagsakService.hentFagsakForBehandling(behandlingId)
+        val personIdent = fagsak.hentAktivIdent()
+        val feilutbetaltePerioderDto = lagFeilutbetaltePerioderDto(behandlingId)
+        val navEnhet = arbeidsfordelingService.hentNavEnhet(personIdent)
+        val request = ForhåndsvisVarselbrevRequest(varseltekst = varseltekst,
```

Comment:
Bør sånt ligge i en service forresten ? (requestobjekter / dtomapping, etc)

Reply:
Hei :) Vi Supplerende Stønad jobber også med tilbakekreving om dagen! Spennende å se hvordan andre team har løst det.

Enig med Trond her. I Supplerende Stønad (https://github.com/navikt/su-se-bakover/tree/master/client/src/main/kotlin/no/nav/su/se/bakover/client) prøver vi å sende domeneobjektene til "klientlaget" (utgående infrastruktur) for så å la det ta seg av mapping og eie statiske variable som domenet ikke bryr seg om. I noen tilfeller kan det være behov for å lage spesifikke domeneobjekter for en spesifikk klient, men da lar vi domenet eie de, og ikke klienten.